boot/seal.go

```diff
@@ -129,10 +130,12 @@ func fallbackKeySealRequests(key, saveKey secboot.EncryptionKey) []secboot.SealK
 	return []secboot.SealKeyRequest{
 		{
 			Key:     key,
+			KeyName: "ubuntu-data-key",
```

Comment:
should these be ubuntu-recovery-data-key and ubuntu-recovery-save-key?

Reply:
well, the key is the same, it depends what we mean by key-name,  that's why we left it out originally.  Anyway from my POV the name is the name of the bits in the key, same bits same name

Reply:
also here no "-key"